Doc/howto/free-threading-extensions.rst

```diff
@@ -99,7 +99,8 @@ Most of the C API is thread-safe, but there are some exceptions.
 * **Macros**: Accessor macros like :c:macro:`PyList_GET_ITEM` and
   :c:macro:`PyList_SET_ITEM` do not perform any error checking or locking.
   These macros are not thread-safe if the container object may be modified
-  concurrently.
+  concurrently. Similarly, using the object returned by
+  :c:func:`PySequence_Fast` concurrently is not thread-safe.
```

Comment:
`PySequence_Fast` is not a macro (but `PySequence_Fast_GET_ITEM` and others are), so maybe rephrase a bit.

There is [Py_BEGIN_CRITICAL_SECTION_SEQUENCE_FAST](https://github.com/python/cpython/blob/8eebe4e6d02bb4ad3f1ca6c52624186903dce893/Include/internal/pycore_critical_section.h#L39C10-L39C49) to make using the result of `PySequence_Fast` thread-safe, but it is in the internal api, so I think it is fine to leave it out of the documentation.

Reply:
Rephrased a bit - do you find this clearer?